config/http/README.md

```diff
@@ -16,7 +16,10 @@ and then pass the request along.
 The factory used to create the actual server object.
 * *no-websockets*: Only HTTP.
 * *websockets*: HTTP and websockets.
-* *https-example*: An example configuration to use HTTPS directly at the server (instead of at a reverse proxy).
+* *https-no-websockets*: Only HTTPS. Adds 2 new CLI params to set the key/cert paths.
+* *https-websockets*: HTTPS and websockets. Adds 2 new CLI params to set the key/cert paths.
+* *https-example*: An example configuration to use HTTPS directly at the server (instead of at a reverse proxy)
+  by adding the required paths to the config itself.
```

Comment:
I don't understand "required paths"

Reply:
Paths to the key/cert files. But yes will change.